pelicun/db.py

```diff
@@ -2681,3 +2681,207 @@ def find_class_type(entry: str) -> str | None:
     # save the metadata
     with open(target_meta_file, 'w+', encoding='utf-8') as f:
         json.dump(meta_dict, f, indent=2)
+
+
+def create_Hazus_Flood_repair_db(source_file_dir: str, target_data_file: str):
+    """
+    Create a database metadata file for the HAZUS Flood
+    loss functions.
+
+    Parameters
+    ----------
+    source_file_dir: string
+        Path to the directory containing Hazus Flood loss function
+        data files.
+    target_data_file: string
+        Path where the loss function data should be saved. A CSV file
+        is expected.
+
+    """
+
+    source_data = {}
+    for subassembly_type in ('structural', 'inventory', 'contents'):
+        source_file = (
+            f'{source_file_dir}/HazusFloodDamageFunctions_'
+            f'Hazus61_{subassembly_type}.csv'
+        )
+        source_data[subassembly_type] = pd.read_csv(source_file)
+
+    def eda():
```

Comment:
Unfortunately even though I see the file name in your comment, I don't see the commit SHA.
`db.py` was chopped up and moved to the [DamageAndLossModelLibrary](https://github.com/NHERI-SimCenter/DamageAndLossModelLibrary). This part appears to be for the Hazus Hurricane metadata, I think it was removed.

Reply:
If you look at this conversation in a web browser, you should be able to click on the filename in the top left corner and it will take you to a diff view with the comment shown inline. This is not part of the hurricane metadata, but a flood parameter CSV generator script. It's from your first commit in this PR.

Reply:
Thanks, this works great.
I removed the function with this [PR](https://github.com/NHERI-SimCenter/DamageAndLossModelLibrary/pull/13).